i18n/locales/en.ts

```diff
@@ -1192,19 +1192,59 @@ const translations = {
+          'OP_PUSHDATA in Bitcoin script plays a crucial role in facilitating the insertion of arbitrary pieces of data into the blockchain. It is actually a whole category of opcodes that allow for the inclusion of data elements of varying sizes, making Bitcoin’s scripting language more versatile. This opcode is particularly significant in enabling the implementation of various smart contract functionalities and custom transaction types. Given that we know this is supposed to push 71 bytes of data onto the stack can you work out how this opcode might be represented in hex? You can read more about some of these OP_CODES and more <Link href="https://en.bitcoin.it/wiki/Script#Constants" target="_blank" className="underline">here</Link>.',
       },
-      verify_signature: {
-        signature_verification_heading: 'Signature Verification',
-        signature_verification_paragraph_one:
-          'Signature verification is crucial in Bitcoin to confirm that a transaction is authorized by the holder of the private key. In the context of ECDSA, it involves checking that a signature (comprising two numbers, r and s) is valid for a given public key and message. This verification ensures the integrity and authenticity of a transaction.',
+      derive_message_four: {
+        op_checksig_heading: 'OP_CHECKSIG',
+        op_checksig_paragraph_one:
+          'OP_CHECKSIG in Bitcoin script is critically important to ensuring that the correct private key is able to spend a given transaction. In almost every Bitcoin script there will be an OP_CHECKSIG to ensure that the person attempting to spend the bitcoin is able to do so with the given key. You can read more about some of these OP_CODES and more <Link href="https://en.bitcoin.it/wiki/Script#Constants" target="_blank" className="underline">here</Link>.',
+      },
+      derive_message_six: {
+        transaction_parts_heading: 'Transaction Parts',
+        transaction_parts_one:
+          'Version: This version indicates how the the transaction will be organzied.',
+        transaction_parts_two:
+          'Number of inputs: The number of inputs in theis transaction.',
+        transaction_parts_three:
+          'Hash on input#0: This is the hash of the tx corresposing to input#0 subsequent inputs will go after part six.',
+        transaction_parts_four:
+          'Index of input#0: This is the index of the input#0 from the funding transaction as it could have many outputs.',
```

Comment:
```suggestion
          'Index of input#0: This is the index of the input#0 from the transaction that created it. This is necessary since the transaction could have many outputs.',
```

In chapter 10 we get into Lightning and the term "funding transaction" is used. While I like using "funding transaction" in this context, I don't want to confuse readers.